profiler/lib.profiler/build.xml

```diff
@@ -34,6 +34,10 @@
         <unzip src="external/profiler-external-binaries-ASF-5273a10f9471d5cf60ed894dc26ec0e59e71cc24.zip"
                overwrite="true"
                dest="${profiler.cluster}"/>
+        <!-- AL2 licensed binaries, newer build from ASF code -->
+        <unzip src="external/profiler-external-binaries-1-24aefa9.zip"
+               overwrite="true"
+               dest="${profiler.cluster}"/>
```

Comment:
```suggestion
        <unzip src="external/profiler-external-binaries-1-24aefa9.zip"
               overwrite="true"
               dest="${profiler.cluster}">
            <!-- Only extract lib folder (licenses, notices are covered by project wide license/notice) -->
            <patternset>
                <include name="lib/**"/>
            </patternset>
        </unzip>
```

Reply:
Yes, I was intending to use `patternset` if we wanted to just use the updated mac binaries.  I'll do this change, although those files don't make it through to the IDE binary anyway.